app/views/employee/employee.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="container">
+
+	<h3 class="page-header">Employees</h3>
+
+	<div class="float-left">
+		<table>
+			<tr>
+				<th>No.</th>
+				<th>Employee Name</th>
+				<th>Employee Surname</th>
+			</tr>
+			<tr>
+				<td>1</td>
+				<td>Daniel</td>
+				<td>Steenberg</td>
+			</tr>
+			<tr>
+				<td>2</td>
+				<td>Geoffrey</td>
+				<td>Rhodes</td>
+			</tr>
+		</table>
+	</div>
+
+	<div class="float-right">
+		<h2>Search Bar</h2></br>
+		<form class="search-form">
+  			Enter Search item here:<br>
+  		<input type="text" name="username"><br>
+		</form>
+		<input class="button" type="button" onclick="alert('Search')" value="Search">
+	</div>
+</div>
```

Comment:
You'll have to refactor this at a later stage to have a show page that pulls the relevant employee details based on an ID. This will affect your controller method and your routes.